PULL_REQUEST_TEMPLATE.md

```diff
@@ -43,6 +43,7 @@ Addresses #*PR# HERE*
 - [ ] Are there any new interactions that need to be added to the [QA Sheet](https://docs.google.com/spreadsheets/d/1HF4Gy6rb_BLbZoNkZEWZonKFBqPyVEiQq4Ve6XgIYmQ/edit#gid=0)?
 - [ ] Are there opportunities for using Google Analytics here (if applicable)?
 - [ ] If the Pipfile has been changed, is the updated Pipfile.lock file also included in this PR?
+- [ ] If the changes add any new endpoints, are there any new probers for those endpoints?
```

Comment:
I think we want to avoid requiring probers for all new endpoints yet. Maybe something like "If this adds a critical new endpoint, is there a prober for this endpoint?"

Reply:
I'm not super happy with that language, BTW, mostly want to take things a bit slow with probers to give us more time to acclimate to them and devise best practices.